app/mailers/notification_mailer.rb

```diff
@@ -29,7 +29,7 @@ def meeting_reminder(meeting, member)
   end
 
   def notification_email(recipientid, data)
-    data = HashWithIndifferentAccess.new(JSON.parse(data))
+    data = ActiveSupport::HashWithIndifferentAccess.new(JSON.parse(data))
```

Comment:
Rails 5.1 soft deprecated the use of `HashWithIndifferentAccess` without the `ActiveSupport::` namespace.